mantidproject/mantid
# Add scripts for new versions of tomographic reconstruction tools

Fixes #12809.

Adds scripts that can run reconstruction tools remotely or locally. They use packages added in previous PRs (#14538) that are tested but these reconstruction scripts will only work on machines that are setup with the required tools (SCARF, the IMAT analysis machine, my development machine).

**To test**: 
- code review, check documentation, all tests should pass.
- Pylint should not report any new warnings (didn't locally, let's see...)

This is part of #14155 and related to #14384.
The system test should be extended further for better coverage of the new functionality, but that will have to wait for another issue.
